modules/learn/pages/data/scopes-and-collections.adoc

```diff
@@ -77,12 +77,32 @@ The default scope cannot be dropped.
 The default collection _can_ be dropped, by means of either the Couchbase CLI, or the REST API.
 Once dropped, the default collection cannot be recreated.
 
-== &#95;system Scope, and its Collections
+== The `_system` Scope, and its Collections
 
 In Couchbase Server Version 7.6+, in each user-created or sample bucket, a `&#95;system` scope is created and maintained by default.
 This scope contains collections used by Couchbase services, for service-specific data.
-The `&#95;system` scope and its collections (including `mobile` and `query`) are _not_ for the use of administrators; and should _not_ be accessed in any way by users or user-created applications.
-The `&#95;system` scope and its collections _cannot_ be dropped.
+The scope and its collections _cannot_ be dropped.
+
+WARNING: Users should NOT rely on the content of those collections.
+Their structure is subject to change without warning.
+
+For reference, the collections are as follows:
+
+`_eventing`::
```

Comment:
@istvano @osfameron 
_eventing and _transaction system collection are not present in 7.6.0. Please remove mention of them. thanks.